bitnami/kafka/templates/statefulset.yaml

```diff
@@ -267,9 +267,15 @@ spec:
               value: {{ default "" .Values.auth.tls.endpointIdentificationAlgorithm | quote }}
             - name: KAFKA_TLS_CLIENT_AUTH
               value: {{ ternary "required" "none" (or (eq (include "kafka.externalClientProtocol" . ) "mtls") (eq .Values.auth.clientProtocol "mtls")) | quote }}
-            {{- $tlsPassword := .Values.auth.tls.password }}
             - name: KAFKA_CERTIFICATE_PASSWORD
-              value: {{ default "" $tlsPassword | quote }}
+              {{- if .Values.auth.tls.passwordSecret }}
```

Comment:
We normally call this parameter existingSecret. In order to maintain certain consistency between charts, could you change it?

Reply:
Hi @javsalgar, sure I can change it. I used `passwordSecret` instead of `existingSecret` because there is already a parameter `existingSecrets` in this chart to provide certificates and it could be confusing. If it is ok for you, I will change.

Reply:
I'd rather continue with the existingSecret approach as it will also help our testing automation

Reply:
Done!